dist/main.js

```diff
@@ -0,0 +1,88 @@
+
+const togglePopup = function(){
+    document.getElementById("popup-1").classList.toggle("active");
+}
+
+const getTeamPlayers = function() {
+    const teamName = $("#nbaTeam").val();
+    $.get(`teams/${teamName}`, function (data) {
+        const source = $('#store-template').html();
+        const template = Handlebars.compile(source);
+        const newHTML = template({data});
+        $('.results').empty().append(newHTML);
+    })
+}
+
+const dreamTeamPlayers = function () {
+    $.get(`dreamTeam`, function (data) {
+        const source = $('#store-template-dreamTeam').html();
+        const template = Handlebars.compile(source);
+        const newHTML = template({data});
+        $('.results').empty().append(newHTML);
+    })
+}
+
+// When you click on any player's image, it will display the player's stats over the image.
+$('.results').on('click', '.playerImg', function(){
+    const player = $(this).closest('.player').find('.playerName').text()
+    $.get(`/playerStats/${player}`, function (data) {
+        $('.title').empty().append(player)
+        if(data.length == 0){ // if no info exist
+            document.getElementById("popup-1").classList.toggle("active");
+            $('.playerSt').empty().append('no information');
+        }
+        else{
+            const source = $('#store-template-playerStats').html();
+            const template = Handlebars.compile(source);
+            const newHTML = template({data});
+            document.getElementById("popup-1").classList.toggle("active");
+            $('.playerSt').empty().append(newHTML);
+        }
+    })
+})
+
+// When you click on any player's name, it will add the player to the dreamTeam
+$('.results').on('click', '.playerName', function(){
+    const name = $(this).closest('.player').find('.playerName').text()
```

Comment:
instead of taking info from the DOM you should do a get request to your server with the player last name and first name then return the data using your server.